src/index-dev.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import App from './containers/App';
+import createStore from './state/createStore';
+import * as actions from './state/actions';
+import './styles/index.scss';
+
+const store = createStore();
+const devConfig = {
```

Comment:
Maybe not a blocker, but it might be good to allow this to be overridden with a local file on the dev's machine so people aren't updating this config all the time in PRs

Reply:
agreed.  I think that the devConfig should use ` .env`

Reply:
implemented `.env`.  If the custom config key (REACT_APP_LOCAL_MIRADOR_CONFIG) does not exist, falls back to `defaultAppConfig`.  These config files are `json`.

Here is the method.
https://github.com/ProjectMirador/mirador/blob/7a739167694684e2b94180a34ebffbca05584ba2/src/index-dev.js#L11-L19